src/wovnio/wovnphp/Utils.php

```diff
@@ -11,7 +11,7 @@ class Utils
     // will return the store and headers objects
     public static function getStoreAndHeaders(&$env)
     {
-        $file = DIRNAME(__FILE__) . '/../../../../wovn.ini';
+        $file = isset($_ENV['WOVN_CONFIG']) ? $_ENV['WOVN_CONFIG'] : DIRNAME(__FILE__) . '/../../../../wovn.ini';
```

Comment:
This means that user should set absolute path to $_ENV['WOVN_CONFIG'], right?
I want to add description about absolute path to README.

Reply:
Added https://github.com/WOVNio/WOVN.php/pull/111/files#diff-4b29e98dc086ca7ee0e534c9eb921739R456